lib/controller.js

```diff
@@ -22,21 +26,22 @@ Tessel.list = function(opts) {
     // When a Tessel is found
     seeker.on('tessel', function displayResults(tessel) {
       var note = '';
-
       // Add it to our array
       foundTessels.push(tessel);
-
       // Add a note if the user isn't authorized to use it yet
       if (tessel.connection.connectionType === 'LAN' && !tessel.connection.authorized) {
         note = '(USB connect and run `tessel provision` to authorize)';
       }
-
```

Comment:
@johnnyman727 yes I did many times (also at other code parts) but the linter automatically edited this many times. So I gave up to do this.